en/components/input.md

```diff
@@ -101,67 +101,65 @@ Data binding are specified by using curly brace syntax, example: {isAdmin}. Text
+When supplied the `🕹️Event` property is used to create a method in the component TypeScript and add an Angular click signature in the HTML for the input’s icons. When supplied the event must be specified using the curly braces format: {onEventName}.
 
 ### DataProperty
 
-When supplied, the `🕹️DataProperty` value is used to set up two-way data binding to the Input Group formControlName property. The `🕹️DataProperty` is optional. When provided, it will be the name of the property on the data object specified by the model object name provided in the generation request.
-
-### Input
+When supplied, the `🕹️DataProperty` value is used to set up a two-way data binding using Angular Reactive Forms to the input value property. The `🕹️DataProperty` is optional. When provided, it will be the name of the property on the data object specified by the model object name provided in the generation request.
 
-Determines the type of styling applied to the Input Group. When set to None, no control will be rendered. The Time Picker and Date/Calendar do not support this. Box, Border, Line, and Searchbar do not have this property as their base type are the values of this field.
+### Inputs (Except Searchbar)
 
-### Variant
+#### Layout
 
-Determines if the theme is light or dark. The Time Picker and Date/Calendar do not support this. If set to None, the control will not be rendered.
+Determines the layout of the Input Group. When set to display Prefix and/or Suffix, text or icons can be displayed at the corresponding ends of the Input.
 
-### State
-
-If set to None, the control will not be rendered.
+#### Label
 
-### Layout
+Determines the text to be displayed in the Label.
 
-Determines the layout of the Input Group. When set to display Prefix and/or Suffix, text or icons can be displayed at the corresponding ends of the input. The Searchbar instead has a Left and Right Icons that can be turned on/off individually and does not support text in the Prefix/Suffix. The Time Picker and Date/Calendar do not support this. When set no None, the control will not be rendered.
+#### Input Text
 
-### Label
+Determines the text to display in the Input value. If the DataProperty is set this is ignored. In the Line, Box, or Border Inputs if this field contains only `*` characters the type of the input will be set to password.
 
-Determines the text to be displayed in the Label
+#### Help Text
 
-### Input Text
+Determines the text to be displayed in the Hint of the Input Group. The Time Picker and Date/Calendar do not support this.
 
-Determines the text to display in the Input value. If the `🕹️DataProperty` is set, this is ignored.
+### Searchbar
 
-### Help Text
+The Searchbar has Left and Right Icons that can be turned on/off individually and each have an Event that can be assigned. The Searchbar has a Search Hint that displays in the label of the Searchbar.
```

Comment:
I'm not an English major, but this sounds weird.  I think it should be "each has" because "each" refers to "each icon" and you would say "each icon has".